Gemfile

```diff
@@ -113,6 +113,6 @@ gem 'ng-rails-csrf'
 # LETSENCRYPT
 # Until the new API calls are generally available, you must manually specify my fork
 # of the Heroku API gem:
-gem 'platform-api', github: 'jalada/platform-api', branch: 'master'
```

Comment:
I'm guessing this branch was merged? If so, can you remove the comment (but leave the # LETSENCRYPT line to clarify what it's for?)